### ISSUE TITLE
Sitecore 8.2 Error

### ISSUE THREAD

**Issue by JakeDK** [BDFurlong/AddChildItemRule#1]
Hej Bryan

Awesome plug-in, this is a really timesaver but I'm getting an error in Sitecore 8.2
<img width="645" alt="screen shot 2017-06-07 at 15 50 10" src="https://user-images.githubusercontent.com/1118844/26881911-2bf1c78e-4b99-11e7-9929-ebcac63a0a4b.png">

<img width="1435" alt="screen shot 2017-06-07 at 15 50 35" src="https://user-images.githubusercontent.com/1118844/26881913-2c6fb978-4b99-11e7-85c9-4efee00250b2.png">

Best Regards
Jakob

**Comment by BDFurlong** [comment]
Can you verify the AddChildItemRule.Cs.Rules.dll is in your bin?

**Comment by BDFurlong** [comment]
er sorry I mean AddChildItemRule.Sc.Rules.dll

**Comment by JakeDK** [comment]
Looks like the AddChildItemRule.Sc.Rules.dll is missing. Where can I find the .dll?

**Comment by BDFurlong** [comment]
Did you install this with a package?

**Comment by JakeDK** [comment]
Yes I installed it as a package, so it should generate the .dll.
